### DISCUSSION TITLE
ERROR: error pulling image configuration: read tcp 192.168.1.23:39256->104.18.122.25:443: read: connection reset by peer

### DISCUSSION THREAD

**Discussion by chrispi68** [teslamate-org/teslamate#1548] [Getting started]
When I run "sudo docker-compose up" I get an error message:

christoph@dockerhost1:~$ sudo docker-compose up
Pulling grafana (teslamate/grafana:latest)...
latest: Pulling from teslamate/grafana
801bfaa63ef2: Pull complete
efdb3434c59e: Pull complete
8cbdb3f56d34: Pull complete
34f82d4bd2ec: Downloading [==========================>                        ]  4.344MB/8.342MB
af445b3382af: Download complete
8aab09bbec8e: Download complete
9e81c23e3db5: Download complete
dd5aa30cd324: Download complete
6b510584a5b7: Download complete
408932f8a13e: Download complete
94d0429c7909: Download complete
4b679a46dd43: Download complete
394292bdf9e2: Download complete
a6391219be16: Download complete
5ad15d91b4ca: Download complete
06dd69662546: Download complete
ERROR: error pulling image configuration: read tcp 192.168.1.23:59282->104.18.123.25:443: read: connection reset by peer

I am using a fresh installation of Ubuntu 20.04.1 Server with latest updates.

Any ideas?

Best regards
Christoph

**Comment by adriankumpf** [comment]
This is an issue with docker itself, see for example https://github.com/moby/moby/issues/27734 

Try to upgrade docker and docker-compose to a newer version.
